app/build.gradle

```diff
@@ -187,6 +187,9 @@ dependencies {
     // Logging with Logcat
     implementation 'com.squareup.logcat:logcat:0.1'
 
+    // Swipe to refresh
+    implementation "com.google.accompanist:accompanist-swiperefresh:0.27.0"
```

Comment:
Nowadays, the swipe refresh layout is contained in Compose already (I think starting at 1.3.0)

Reply:
Cool, I will refactor.